src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -776,12 +776,12 @@ Color.white, new Procedure("condition4"),
 			armor.creativeTab = new TabEntry(modElement.getWorkspace(),
 					getRandomDataListEntry(random, ElementUtil.loadAllTabs(modElement.getWorkspace())));
 			armor.armorTextureFile = "armorTexture";
-			armor.maxDamage = 12;
+			armor.maxDamage = 15;

```

Comment:
Please revert those changes, old values were ok:

> test values should deviate for UI defaults, so tests actually test if fields change properly

Reply:
I had noticed that it was wrong

Reply:
This is not wrong. Test values are arbitrary values selected only to differ from UI values and have nothing to do with Minecraft values. 

Please, revert this file

Reply:
Ok

Reply:
Done

Reply:
There are still changes in this file that should _not_ be there

The purpose of this file is _not_ to match defaults, values here should _not_ match default values by its design

Reply:
![image](https://github.com/MCreator/MCreator/assets/16374228/7091b300-e1ca-4514-8047-bb1a4d8ed494)

Please respect our rules and do not mark as resolved by yourself.....

Reply:
This was **_not_** resolved

Reply:
Done

Reply:
> There are still changes in this file that should _not_ be there
> 
> The purpose of this file is _not_ to match defaults, values here should _not_ match default values by its design

I didn't understand to remove all the changes from that file but only from the line you highlighted